.github/workflows/test.yml

```
@@ -68,3 +68,37 @@ jobs:
         env:
           COVERALLS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         if: runner.os == 'Linux'
+  # Build and run frogbot current changes for visual sanity check.  
+  sanity:
+    needs: tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17.x
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Build Frogbot
+        run: buildscripts/build.sh
+      - name: Scan
+        env:
+          # [Mandatory] JFrog platform URL
+          JF_URL: ${{ secrets.FROGBOT_URL }}
+
+          # [Mandatory if JF_USER and JF_PASSWORD are not provided] 
+          JF_ACCESS_TOKEN: ${{ secrets.FROGBOT_ACCESS_TOKEN }}
+          
+          # [Manadatory] The GitHub token automatically generated for the job
+          JF_GIT_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

I think we don't need these explanations in an internal sanity test
```suggestion
          JF_URL: ${{ secrets.FROGBOT_URL }}
          JF_ACCESS_TOKEN: ${{ secrets.FROGBOT_ACCESS_TOKEN }}
          JF_GIT_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```